website/docs/index.html.markdown

```diff
@@ -122,6 +177,10 @@ The following arguments are used to configure the VMware vCloud Director Provide
 * `password` - (Required) This is the password for vCloud Director API operations. Can
   also be specified with the `VCD_PASSWORD` environment variable.
   
+* `token` - (Optional; *v2.6+*) This is the authorization token that can be used
+   instead of username and password. When this is set, username and password will not
+    be used and may be left empty, or may contain other remarks.
```

Comment:
What other remarks? Need to be more specific.

Reply:
Something like 
```
  user     = "USING TOKEN"
  password = "USING TOKEN"
  token    ="d1e2a3d4b5e5e6f7"
```
This way, if there are tools that extract such fields, they will show the remarks instead of the real value.

Reply:
Then suggesting this rephrase:

```
When `token` is set, username and password will be ignored, but should be left in configuration either empty or with any custom values.
```

Reply:
done